src/main/java/org/apache/commons/csv/CSVParser.java

```diff
@@ -765,7 +805,14 @@ CSVRecord nextRecord() throws IOException {
         final long startCharPosition = lexer.getCharacterPosition() + this.characterOffset;
         do {
             this.reusableToken.reset();
-            this.lexer.nextToken(this.reusableToken);
+            // https://issues.apache.org/jira/browse/CSV-147
+            try {
+                this.lexer.nextToken(this.reusableToken);
+            } catch (IOException ioe) {
+                String errorMessage = "An exception occurred while parsing the CSV content. Issue in line: "
+                        + this.lexer.getCurrentLineNumber() + ", position: " + this.lexer.getCharacterPosition();
```

Comment:
Simply message: "Exception parsing at line N, column C."

Reply:
Done.
However, column and position are two different things. Therefore kept the position as it is.
lexer is not having any method to access the column as I see.

Reply:
![image](https://github.com/apache/commons-csv/assets/5607766/52890cb4-018c-448e-94bd-8e144eca6fc3)

Reply:
> Done. However, column and position are two different things. Therefore kept the position as it is. lexer is not having any method to access the column as I see.

I meant position, sorry.